hack/boilerplate/verify_boilerplate.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
A note here: I didn't make any of these edits myself.  I merely downloaded the latest copy of the script from kubernetes/repo-infra